Source/PluginEditor.cpp

```diff
@@ -171,5 +181,5 @@ void SubsynthAudioProcessorEditor::setGainStyle()
     gainSlide.setValue (-25.0);
     gainSlide.setTextBoxStyle (juce::Slider::NoTextBox, false, 0, 0);
     gainSlide.setTextValueSuffix (" dB");
-    adsrSliders.setBounds (220, 55, 400, 100);
+    adsrSliders.setBounds (0.3298 * width, 0.0647 * width, 0.4706 * width, 0.1176 * width);
```

Comment:
I think this is a duplicate? It's already called in `resized()` above, I don't think it needs to also be here.

Reply:
Its also a bit weird to have adsrSlider sizing under the `setGainStyle` method?

Reply:
I wasn't sure why it was there, and it actually took me a while to track it down.  It was causing a bug where the ADSR sliders were starting in the wrong spot but the moment you'd resize the window they'd snap into the correct place.  I'll remove it.